include/oneapi/dpl/internal/serial_algorithm_impl.h

```diff
@@ -37,6 +37,8 @@ for_each_n(_InputIterator __first, _Size __n, _Function __f)
     }
     return __first;
 }
+#    else
```

Comment:
Previously these lines were erroneously deleted

Reply:
Other words, we have C++17 configuration where C++ library has not `std::for_each_n`, right?